docs/source/external_exporters.rst

```diff
@@ -5,26 +5,62 @@ Using external exporters
 
 .. versionadded:: 4.2
 
-    You can now use the ``--to`` flag to use custom export formats defined in other
-    libraries.
+    You can now use the ``--to`` flag to use custom export formats defined outside nbconvert.
 
 
 The command-line syntax to run the ``nbconvert`` script is::
 
-  $ jupyter nbconvert --to FORMAT notebook.ipynb
+  jupyter nbconvert --to FORMAT notebook.ipynb
 
 This will convert the Jupyter document file ``notebook.ipynb`` into the output
 format designated by the ``FORMAT`` string as explained below.
 
 A few built-in formats are available by default: `html`, `pdf`,
 `script`, `latex`. Each of these has its own _exporter_ with many configuration
-options that can be extended. Having the option to point to a different _exporter_ 
-allows authors to create their own fully customized templates. 
+options that can be extended. Having the option to point to a different _exporter_
+allows authors to create their own fully customized templates or export formats.
 
-A custom _exporter_ must be a globally importable python object. We recommend that
-these be distributed as python libraries.
 
-To export using an exporter from an external library, use the `fully qualified`
+A custom _exporter_ must be an importable Python object. We recommend that
+these be distributed as Python libraries.
+
+.. _entrypoints:
+
+Entry points
+------------
+
+Additional exporters may be registered as `entry points`_.
+nbconvert uses the ``nbconvert.exporters`` entry point to find exporters from any package you may have installed.
+
+If you are writing a Python package that provides an exporter,
+you can register them in your package's :file:`setup.py`:
+
+.. sourcecode:: python
+
+    setup(
+        ...
+        entry_points = {
+            'nbconvert.exporters': [
+                'foo = mymodule.FooExporter',
```

Comment:
Shouldn't it `mymodule:FooExporter` with a colon instead of a dot? See the Python packaging docs: https://packaging.python.org/en/latest/distributing/#entry-points